### ISSUE TITLE
Copying goes to the wrong clip-board

### ISSUE THREAD

**Issue by p-h** [xournalpp/xournalpp#1026]
On Linux, Xournalpp writes to the wrong clip-board when copying. The clipboard used is [PRIMARY and not CLIPBOARD](https://mutelight.org/subtleties-of-the-x-clipboard). It can only be pasted using middle click and not CTRL-V

**Comment by andreasb242** [comment]
Confirmed, but this affects only the Text editor, graphics are copied correctly!

**Comment by edo0** [comment]
The behaviour I am currently noticing in my installation (build of April 20), is that if I paste text in a textbox, the pasted text will not appear immediately, but only after I click outside of the textbox (or anyway unfocus said textbox)

**Comment by JJones780** [comment]
@edo0  I can confirm that I get the delayed paste behaviour as well... I didn't spot an immediate fix though.  Please create a new issue report for it.

**Comment by edo0** [comment]
> @edo0 I can confirm that I get the delayed paste behaviour as well... I didn't spot an immediate fix though. Please create a new issue report for it.

Opened a new ticket (1103). Sorry for having intruded in  this conversation!

**Comment by JJones780** [comment]
@edo0  It's great that you noted a new issue.  Just didn't want it to get lost when this issue gets closed. Thanks for creating a new separate issue.

**Comment by Technius** [comment]
This issue can be closed now.
